kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -47,9 +47,9 @@ public void testOddNumbersExterminatorNormalList() {
         ArrayList<Integer> list = TestingMain.initNumbers();
         ArrayList<Integer> resultList = new OddNumbersExterminator().exterminate(list);
         //When
-        Integer checksum = new OddNumbersExterminator().checkList(resultList);
+        boolean evenValidator = new OddNumbersExterminator().isEven(resultList);
        //Then
-        Assert.assertTrue(checksum % 2 == 0);
+        Assert.assertEquals(true, evenValidator);
```

Comment:
Wiesz co. Ja widzę to tak w testach, że przygotowuję jakieś scenariusze testowe (dla różnch przypadków) i je sprawdzam. Testy trochę się różnią od samego oprogramowania, które piszemy.
Np.
`// given`
`List<Integer> list = Arrays.asList(1, 2);`
`List<Integer> expected = Arrays.asList(2);`
`// when`
`List<Integer> actual = OddNumbersExterminator.exterminate(list);    // uwaga: metoda statyczna!`
`// then`
`Assert.assertEquals(actual, expected);`